src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -66,6 +66,47 @@ If `instrument.js` displays in your console while debugging, add Sentry to your
 
 ## Dealing with Ad-Blockers
 
+### Using the `tunnel` Option
+
+Starting with version `6.5.0` of the JavaScript SDK, you can use the `tunnel` option to tell the SDK to deliver events to the configured URL, instead of using the DSN. This allows the SDK to remove `sentry_key` from the query parameters, which is one of the main reasons ad-blockers prevent sending events. This option also stops the SDK from sending preflight requests, which was one of the requirements that necessitated sending the `sentry_key` in the query parameters.
```

Comment:
```suggestion
Starting with version `6.7.0` of the JavaScript SDK, you can use the `tunnel` option to tell the SDK to deliver events to the configured URL, instead of using the DSN. This allows the SDK to remove `sentry_key` from the query parameters, which is one of the main reasons ad-blockers prevent sending events. This option also stops the SDK from sending preflight requests, which was one of the requirements that necessitated sending the `sentry_key` in the query parameters.
```